build/BuildSteps.cs

```diff
@@ -154,6 +154,11 @@ public static void BuildExtensionsBundle(BuildConfiguration buildConfig)
                 publishCommandArguments += $" /p:PublishReadyToRun=true";
             }
 
+            if (buildConfig.SuppressTfmSupportBuildWarnings)
+            {
+                publishCommandArguments += " /p:SuppressTfmSupportBuildWarnings=true";

```

Comment:
This likely would have unintended consequences as it would suppress all such warnings, the ones we want and the ones we don't. Instead, we should make the dependency conditional on TFM in the extension package.

Reply:
Can you elaborate what you mean by "make the dependency conditional on TFM in the extension package"? I don't think I quite understand.